vms/evm/contract_message_test.go

```diff
@@ -72,6 +72,7 @@ func TestUnpack(t *testing.T) {
 			input, err := hex.DecodeString(testCase.input)
 			require.NoError(t, err)
 
+			mockLogger.EXPECT().Warn(gomock.Any(), gomock.Any()).Times(testCase.errorLogTimes)
```

Comment:
This had to be added to account for the changes discussed [here](https://github.com/ava-labs/awm-relayer/pull/153/files#r1464206597)